hack/bazel-test.sh

```diff
@@ -3,6 +3,12 @@ set -e
 source hack/common.sh
 source hack/config.sh
 
+if [ "${CI}" == "true" ]; then
```

Comment:
Test results were still cached. I think the CI env variable needs to be passed through in the Makefile target.